sources/main.cpp

```diff
@@ -8,25 +8,58 @@
 // SPDX-License-Identifier: MPL-2.0
 
 #include "Context.h"
+#include "Dico.h"
 #include "Log.h"
+#include "Message.hpp"
 #include "Options.h"
 
 #include <DYNError.h>
+#include <boost/filesystem.hpp>
+#include <cstdlib>
 #include <sstream>
 
+static const char* dictPrefix = "DFLMessages_";
+
+static std::string
+getMandatoryEnvVar(const std::string& key) {
+  char* var = getenv(key.c_str());
+  if (var != NULL) {
+    return std::string(var);
+  } else {
+    // we cannot use dictionnary errors since they may not be initialized yet
+    LOG(error) << "Cannot find environnement variable " << key << " : please check runtime environement" << LOG_ENDL;
+    std::exit(EXIT_FAILURE);
+  }
+}
+
 int
 main(int argc, char* argv[]) {
   try {
     DYN::Trace::init();
     dfl::common::Options options;
 
     if (!options.parse(argc, argv)) {
-      LOG(error) << options << LOG_ENDL;
+      LOG(info) << options << LOG_ENDL;
       return 0;
     }
     dfl::common::Log::init(options);
 
+    std::string locale = getMandatoryEnvVar("DYNAFLOW_LAUNCHER_LOCALE");
+    std::string dictDir = getMandatoryEnvVar("DYNAFLOW_LAUNCHER_DICTS");
```

Comment:
This mandatory environment variable is not specified in the README

Reply:
OK